Formula/fdroidserver.rb

```diff
@@ -15,25 +15,35 @@ class Fdroidserver < Formula
 
   depends_on :python3
   depends_on "freetype"
-  depends_on "jpeg"
```

Comment:
openjpeg provides JPEG2000 support, which I think is non-overlapping with libjpeg, so I'd provide both. pillow oughta build fine without JPEG2000 support, though, so who knows.